app/constants/license.ts

```diff
@@ -8,7 +8,7 @@ export default {
         Starter: 'starter',
         Professional: 'professional',
         Enterprise: 'enterprise',
-        Premium: 'premium',
+        EnterpriseAdvanced: 'advanced',
```

Comment:
What about self hosted below?

Reply:
I see only one use, just for SelfHostedProducts.STARTER, others are all unused and not needed, including for new SKU.

Reply:
yeah, I'm not sure why is that there, perhaps we need to reach out to the person who added it or something, I'm not even sure what it is used for, but at the end of the day my comment was mostly about consistency other than usage

Reply:
@M-ZubairAhmed can you share why do we need those? I see you added them as part of #7663 and in that PR, the only use was of `SelfHostedProducts.STARTER` and till now we don't have any other use of that or other keys.

Reply:
If its not used you can remove it i have added that for consistency.

Reply:
Okay, so for this PR, I'll keep it as it, and we're find just adding the constant for short sku name.

Reply:
@enahum we should be good to go for this. I've created https://mattermost.atlassian.net/browse/MM-63963 to track the removal of unused constants.